agent/config/builder.go

```diff
@@ -2588,3 +2575,38 @@ func (b *builder) buildTLSConfig(rt RuntimeConfig, t TLS) (tlsutil.Config, error
 
 	return c, nil
 }
+
+func (b *builder) parsePrefixFilter(telemetry *Telemetry) ([]string, []string) {
+	var telemetryAllowedPrefixes, telemetryBlockedPrefixes []string
+
+	// TODO(FFMMM): Once one twelve style RPC metrics get out of Beta, don't remove them by default.
+	operatorPassedOneTwelveRPCMetric := false
+	oneTwelveRPCMetric := *telemetry.MetricsPrefix + "." + strings.Join(middleware.OneTwelveRPCGauges[0].Name, ".")
```

Comment:
Should this end with a `.` suffix?

Reply:
hey matt, I don't think I understand this. 

`oneTwelveRPCMetric` will look something like

```
oneTwelveRPCMetric = "consul" + "." + "rpc.server.call" = "consul.rpc.server.call"
```

> It looks as though there is one metric that can have many labels.

Yeah, for 1.12, there will be only `consul.rpc.server.call` with labels {`method`, `errored`, `request_type`, `rpc_type` }